src/runez/__main__.py

```diff
@@ -47,12 +49,13 @@ def cmd_diagnostics():
     """Show system diagnostics sample"""
     parser = runez.cli.parser()
     parser.add_argument("--border", default="colon", choices=NAMED_BORDERS, help="Use custom border.")
-    parser.add_argument("--pyenv", default="~/.pyenv", help="Pyenv folder to scan for python installations.")
+    parser.add_argument("--pyenv", default="PATH", help="Colon separated locations to examine for python installations")
```

Comment:
Help text isn't quite helpful enough.  Does the default imply that it reads the env var?  Can I specify any var?

Reply:
It's just the plain string `PATH` (that represents the env var `PATH`), no other env var is supported.
Possibilities are explained here: https://github.com/codrsquad/runez/blob/main/src/runez/pyenv.py#L919-L923